### PR TITLE
chore: using util lib to exectue cmd

### PR DESCRIPTION
**Pull request by mayocream**
### What this PR does / why we need it:
I was reading APISIX code, and found some code could be simplified or reformat.
- simplify type check
- reformat indent
- remove unnecessary local var

This PR didn't solve any problems though.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
None.
* [ ] Have you added corresponding test cases?
No addition features.
* [ ] Have you modified the corresponding document?
No docs related.
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/core/table.lua [hunk 1]
```diff
@@ -61,9 +61,7 @@ end
 
 
 function _M.try_read_attr(tab, ...)
-    local count = select('#', ...)
```
Comment (spacewander) [inline apisix/core/table.lua:64]: Changes below are quite trivial. We don't accept such change which breaks the `git blame`.
Please remove them.

File: apisix/cli/ops.lua [hunk 2]
```diff
@@ -631,11 +631,10 @@ local function start(env, ...)
     pid = tonumber(pid)
     if pid then
         local lsof_cmd = "lsof -p " .. pid
-        local hd = popen(lsof_cmd)
-        local res = hd:read("*a")
+        local res, err = util.execute_cmd(lsof_cmd)
```
Comment (spacewander) [inline apisix/cli/ops.lua:634]: We only can accept this change which refactors code and use the common way.

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by mayocream** [comment]
https://github.com/apache/apisix/pull/4406#discussion_r649002209
It's alright only merge this part.

**Comment by spacewander** [comment]
Need to fix the lint

**Comment by mayocream** [comment]
Fixed.

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**